pyuntl/untldoc.py

```diff
@@ -14,17 +14,13 @@
 from lxml.etree import iterparse
 from rdflib import Namespace, Literal, URIRef, ConjunctiveGraph
 
-from pyuntl import (UNTL_XML_ORDER, DC_ORDER, ETD_MS_ORDER,
```

Comment:
Can we delete `ETD_MS_ORDER` from `pyuntl/__init__.py`?

Reply:
Will do!